facades/TeraEd/src/main/java/org/terasology/editor/TeraEd.java

```diff
@@ -66,23 +56,42 @@ public void run() {
             logger.warn("Failed to set look and feel to Nimbus", e);
         }
         try {
-            LwjglPortlet lwjglPortlet = new LwjglPortlet();
+            LwjglPortlet portlet = new LwjglPortlet();
 
             PathManager.getInstance().useDefaultHomePath();
 
             engine = new TerasologyEngineBuilder()
-                    .add(new LwjglGraphics())
                     .add(new LwjglTimer())
                     .add(new LwjglAudio())
-                    .add(new LwjglInput())
-                    .add(lwjglPortlet).build();
+                    .add(new AwtInput())
+                    .add(new BindsSubsystem())
+                    .add(portlet).build();
             sceneProperties = new SceneProperties(engine);
```

Comment:
Your game not runs in teraEd because LwjglTime returns 0.0 every time (see StageLoading)
You should initialize GLFW for timer starts counts. (or use different timer)

```suggestion

            if (!GLFW.glfwInit()) {
                throw new RuntimeException("Failed to initialize GLFW");
            }
            sceneProperties = new SceneProperties(engine);
```